source-code/sdk-js/src/helpers/ast.ts

```diff
@@ -0,0 +1,108 @@
+import type { CallExpression, Node, Program } from "estree"
+import { walk } from "estree-walker"
+import { get } from "lodash"
+
+type wrapVariableDeclaration = (
+	sourceAst: Program,
+	searchIdentifier: string,
+	identifier: string,
+) => Promise<Program>
+
+type insertAst = (
+	sourceAst: Program,
+	ast: Node,
+	position:
+		| { after?: never; before: [string, string, ...string[]] }
+		| { after: [string, string, ...string[]]; before?: never },
+) => Promise<Program>
+
+const wrapWithCallExpression = (n: Node, identifier: string) => {
```

Comment:
please don't use single character names for variables. `n` => `node`

Reply:
Will fix